reporting/ugandaemr_reports.md

```diff
@@ -93,3 +93,44 @@ The dashboard shows the avialable reports in the different categories:
 
 ![106A 1B Completed](../.gitbook/assets/106A_1A_download_preserve.png)
 
+## Report Definitions 
+This section describes the cohorts used to build the different types of the reports to provide a basis for validating them 
+
+### TX_NEW
+
+Clients who started ART during the quarter at this facility 
+
+| Includes                                     | Excludes                                  | Notes |
+| -------------------------------------------- | ----------------------------------------- | ----- |
+| Clients with an ART start date in the period |                                           |       |
+|                                              | Patients transferred in during the period |       |
+
+### TX_CURRENT - 90
+
+This the MoH definition which defines Lost to Followup(LTFU) patients as those who have not had a visit in 90 days from the date of the last scheduled appointment 
+
+| Includes                                                     | Excludes                                                     | Notes                                                        |
+| ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| Clients with Baseline ART Regimen                            |                                                              | Have started ART since they have a baseline regimen          |
```

Comment:
@ssmusoke and @dbaluku please include whether this is an AND or an OR for better clarity

Reply:
@solemabrothers What are your recommendations for the language to use?